src/db-access/expenses.ts

```diff
@@ -86,3 +87,78 @@ export async function selectExpenses(
 		})
 		.then(handleDatabaseFullfillment, handleDatabaseRejection);
 }
+
+export async function getSumUnprocessed(): Promise<DatabaseResult<number>> {
+	return database
+		.transaction(async (tx) => {
+			return (await tx.select().from(expensesTable).where(isNull(expensesTable.handlingDate))).reduce(
+				(accumulator, currentValue) => {
+					const moneyParseResult = z.coerce.number().positive().safeParse(currentValue.moneyAmount);
+					if (moneyParseResult.success){
```

Comment:
```suggestion
					if (moneyParseResult.success){
```
Make this if statement into a guard clause